config.py

```diff
@@ -0,0 +1,37 @@
+""" Setting Up the App Configuration"""
+import os
+
+class Config: 
+    SECRET_KEY = os.environ.get('SECRET_KEY') or os.urandom(16)
+    SQLALCHEMY_TRACK_MODIFICATION = False
+    SQLALCHEMY_ECHO = False
+    DEBUG_TB_INTERCEPT_REDIRECTS = False
+    WTF_CSRF_ENABLED = False
+
+    @staticmethod
+    def init_app(app):
+        pass
+    
+class DevelopmentConfig(Config):
+    """ Configuration for Development Environment. """
+    ENV = 'development'
+    DEBUG = True
+    SQLALCHEMY_DATABASE_URI = os.environ.get('DEVELOPMENT_DATABASE') or 'postgresql:///recipe'
+
+class TestingConfig(Config):
+    """Configurations for Testing Environment."""
+    TESTING = True
+    SQLALCHEMY_DATABASE_URI = os.environ.get('TEST_DATABASE') or 'postgresql:///recipe_test'
+
+class ProductionConfig(Config):
+    """Configurations for Production Environment."""
+    ENV = 'production'
+    SQLALCHEMY_DATABASE_URI = os.environ.get('PRODUCTION_DATABASE')
+
+config = {
+    'development': DevelopmentConfig,
+    'testing': TestingConfig,
+    'production': ProductionConfig,
+    'default': DevelopmentConfig
+} 
+
```

Comment:
@Huulamnguyen this code looks familiar. Are you reusing code from somewhere else ? If so you need to cite the source of the piece of code you are re-using. Please use this for reference [https://guides.libraries.uc.edu/citing/code](https://guides.libraries.uc.edu/citing/code).

Reply:
@alexkayabula Yes. I reused a source code. I will describe it clearly in my code base